deltaspike/modules/data/impl/src/main/java/org/apache/deltaspike/data/impl/builder/AnnotatedQueryBuilder.java

```diff
@@ -88,4 +83,12 @@ else if (query.isNative())
         return context.applyRestrictions(result);
     }
 
+    private Class<?> getMethodResultEntityClass(Method m){
```

Comment:
sry, but which original ticket do you mean? [DELTASPIKE-1089](https://issues.apache.org/jira/browse/DELTASPIKE-1089)?